coderd/workspacestats/reporter.go

```diff
@@ -146,11 +147,45 @@ func (r *Reporter) ReportAgentStats(ctx context.Context, now time.Time, workspac
 
 	var errGroup errgroup.Group
 	errGroup.Go(func() error {
-		err := r.opts.StatsBatcher.Add(now, workspaceAgent.ID, workspace.TemplateID, workspace.OwnerID, workspace.ID, stats)
+		start := time.Now()
+		connectionsByProto := json.RawMessage(`[]`)
```

Comment:
I wonder if `[{}]` would be a more accurate representation here? 🤔 Not sure it's necessary though.